# returns the bodyid given its name
string sessionname
int32 sessionid
string name
---
int32 bodyid
